server/monitor.go

```diff
@@ -2377,22 +2377,30 @@ func (s *Server) accountInfo(accName string) (*AccountInfo, error) {
 
 // JSzOptions are options passed to Jsz
 type JSzOptions struct {
-	Account    string `json:"account,omitempty"`
-	Accounts   bool   `json:"accounts,omitempty"`
-	Streams    bool   `json:"streams,omitempty"`
-	Consumer   bool   `json:"consumer,omitempty"`
-	Config     bool   `json:"config,omitempty"`
-	LeaderOnly bool   `json:"leader_only,omitempty"`
-	Offset     int    `json:"offset,omitempty"`
-	Limit      int    `json:"limit,omitempty"`
+	Account     string `json:"account,omitempty"`
+	Accounts    bool   `json:"accounts,omitempty"`
+	Streams     bool   `json:"streams,omitempty"`
+	Consumer    bool   `json:"consumer,omitempty"`
+	Config      bool   `json:"config,omitempty"`
+	LeaderOnly  bool   `json:"leader_only,omitempty"`
+	Offset      int    `json:"offset,omitempty"`
+	Limit       int    `json:"limit,omitempty"`
+	Replication bool   `json:"replication,omitempty"`
 }
 
 type StreamDetail struct {
-	Name     string          `json:"name"`
-	Cluster  *ClusterInfo    `json:"cluster,omitempty"`
-	Config   *StreamConfig   `json:"config,omitempty"`
-	State    StreamState     `json:"state,omitempty"`
-	Consumer []*ConsumerInfo `json:"consumer_detail,omitempty"`
+	Name           string                `json:"name"`
+	Cluster        *ClusterInfo          `json:"cluster,omitempty"`
+	Config         *StreamConfig         `json:"config,omitempty"`
+	State          StreamState           `json:"state,omitempty"`
+	Consumer       []*ConsumerInfo       `json:"consumer_detail,omitempty"`
+	ReplicationLag []*ReplicationLagInfo `json:"replication_lag"`
```

Comment:
Maybe add omitempty like the others just in case?

```suggestion
	ReplicationLag []*ReplicationLagInfo `json:"replication_lag,omitempty"`
```

Reply:
Done! I also added it to the fields of ReplicationLagInfo, though they really shouldn't be empty, I just saw that was the common way.